### ISSUE TITLE
Upgrade to servlet 5 spec

### ISSUE THREAD

**Issue by gkresic** [pippo-java/pippo#625]
Migrate to servlet 5 spec. As of 05.02.2023. Jetty and Tomcat supports it, but Undertow does not.

For reference:

Jetty versions: https://www.eclipse.org/jetty/download.php#version-table
Tomcat verions: https://tomcat.apache.org/whichversion.html#Apache_Tomcat_Versions
Undertow versions: https://undertow.io/downloads.html

Depends on #624

**Comment by gkresic** [comment]
Duplicate of #624
